genesis/src/genesis/dev-albatross-4-validators.toml

```diff
@@ -2,6 +2,8 @@ name = "dev-albatross"
 seed_message = "Albatross DevNet"
 timestamp = "2021-07-15T00:00:00.000+00:00"
 vrf_seed = "e8c7f2f3935da9ca39419aa7d2cc90817245f75e58cc543f2b9478766308e8a50fffccb09e2df3546f5a0c0059d73a506c48fa2b546f15b511d0f7a63f0ee20cd510a87f520e26478bb687ca31a08db8b02921f9a22e32a790c07f16dbdf4501"
+parent_hash = "d7b808129639b27cfc6c3245111db37ba85099496c66ffcb14a6754eb5a1007f"
+parent_election_hash = "264aaf8a4f9828a76c550635da078eb466306a189fcc03710bee9f649c869d12"
```

Comment:
What hashes are these now? Random? Or from Nimiq PoW chain?

Reply:
They are from PoW chain. The parent election hash is block 1's hash and the parent hash is a hash of a recent block. This will be the last block we will be migrating from (for now I picked a recent block e.g. 2732425)